README.md

```diff
@@ -22,8 +22,15 @@ This is the backend server for a web application for tracking the placement acti
     ```sh
     npm install
     ```
+2. Create a .env file in the root directory of the project, and add the following environment variables:
+      ```env
```

Comment:
the alignment of code is bad. Do it patiently @BIJJUDAMA! 😄

Reply:
Oooh ok 
Thanks for the guidance

Reply:
@Ashrockzzz2003 How about now?